EngD/test_config_in_ff_sync.R

```diff
@@ -0,0 +1,20 @@
+code_location <- "C:/Kodingan3/ForestForesight/"
```

Comment:
what is EngD folder? if it is a test it should be in tests folder. also use correct convention for the test file naming, test-...

Reply:
test_config_in_ff_sync.R under EngD folder is not a unittest. 
as I mentioned, I couldn't run a unittest using config file.
This is why I created EngD folder to store all my runner scripts (I had lot other runner scripts in my locals).

Are you OK having a non unit test script like my test_config_in_ff_sync.R under tests/testthat folder?

Reply:
maybe you can just have the runners script on your branch do not push it to the remote? do we need it?

Reply:
Yes, i removed this after having the unittest works with ff_prep_refactored that using config vars